examples/action_dann_lightn/model.py

```diff
@@ -64,7 +65,8 @@ def get_feat_extractor(model_name, num_classes, num_channels):
     """
 
     if model_name == 'I3D':
-        feature_network = i3d(name='rgb_imagenet', pretrained=True)
+        pt_name = 'rgb_imagenet' if num_channels == 3 else 'flow_imagenet'
+        feature_network = i3d(name=pt_name, num_channels=num_channels, pretrained=True)
         # model.replace_logits(num_classes)
         feature_dim = 1024
```

Comment:
Must `feature_dim` be hard-coded? (I am not sure. If it is a convention, you may add a comment to explain to make the code more readable.